admin_guide/limits.adoc

```diff
@@ -65,6 +65,15 @@ spec:
         memory: "100Mi" <13>
       maxLimitRequestRatio:
         cpu: "10" <14>
+    -
+      type: openshift.io/image-size
```

Comment:
I would prefer we create two limit range examples.

The one example that resembles the core upstream Kubernetes object.

Another example that holds just OpenShift types.